hoot-core/src/main/cpp/hoot/core/ops/GeometryModifierOp.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2015, 2016, 2017, 2018, 2019 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+
+#include "GeometryModifierOp.h"
+#include "hoot-core/src/main/cpp/hoot/core/visitors/geometrymodifiers/GeometryModifierAction.h"
+
+// Hoot
+#include <hoot/core/util/ConfigOptions.h>
+#include <hoot/core/util/Factory.h>
+#include <hoot/core/util/MapProjector.h>
+
+namespace bpt = boost::property_tree;
+using namespace boost;
+
+namespace hoot
+{
+
+HOOT_FACTORY_REGISTER(OsmMapOperation, GeometryModifierOp)
+
+const std::string GeometryModifierAction::FILTER_TAG = "filter";
+const std::string GeometryModifierAction::ARGUMENT_TAG = "arguments";
+
+GeometryModifierOp::GeometryModifierOp()
+{
+  _rulesFileName = ConfigOptions().getGeometryModifierRulesFile();
+
+  // get and instantiate available actions
+  std::vector<std::string> availableActionTypes = Factory::getInstance().getObjectNamesByBase(GeometryModifierAction::className());
+
+  LOG_DEBUG( "Available Geometry Modifiers:")
+  for( std::vector<std::string>::iterator it = availableActionTypes.begin(); it != availableActionTypes.end(); it++ )
```

Comment:
For consistency across files and our coding standard, this (and other uses of the parenthesis) should have a space between the `for` and the opening parenthesis, with no space after, nor before the closing one.
```
for (std::vector<std::string>::iterator it = availableActionTypes.begin(); it != availableActionTypes.end(); ++it)
``
Also note the iterator incremented with the prefix++ operator, which, according to our static analysis tools, is the optimal way to increment iterators so it has become our standard.

Reply:
I replaced these ugly things altogether and used the C++11 for(:) loop instead.
Fixed